lib/modules/dosomething/dosomething_global/dosomething_global.features.language.inc

```diff
@@ -33,7 +33,7 @@ function dosomething_global_locale_default_languages() {
     'plurals' => 0,
     'formula' => '',
     'domain' => '',
-    'prefix' => 'es-mx',
+    'prefix' => 'mx',
```

Comment:
if we are changing the prefixes here, should we also change them in the language map strongarm variable?